cmd/egnaddrs/README.md

```diff
@@ -0,0 +1,41 @@
+## egnaddrs
+This tool is used to help debug and test eigenlayer/avs deployments and contract setups.
+
+### How to install
+#### Install from repository
+```bash
+   go install github.com/Layr-Labs/eigensdk-go/cmd/egnaddrs@latest
+```
+
+#### Install from local source
+If you have git cloned eigensdk-go to your machine, navigate to [cmd/egnaddrs](.) directory and run
+```bash
+   go install
+```
+
+### Usage
+
+Currently egnaddrs only supports deriving contract addresses starting from the registry-coordinator address. It then prints the following datastructure:
```

Comment:
can you update the readme to reflect that we can use service manager too.

Reply:
done